components/utils/PlusButton.jsx

```diff
@@ -13,11 +13,11 @@ const ButtonContainer = styled(TouchableOpacity)`
 
 const PlusButton = (props) => {
   const {
-    onPress, title, size, style,
+    onPress, title, size, style, font,
   } = props;
 
   const Buttontext = styled.Text`
-      font-family: 'Montserrat_500Medium';
+      font-family: ${font};
```

Comment:
Im a little unsure as to why font is a prop? Is the font something that might change depending on where it occurs? Is this just a good practice?

Reply:
I added it bc on the mockups the font for the add set button is roboto not montseratt. I didn't use it bc I thought Montseratt looks better, but I left it in case we want to change it in the future. Can probably remove it it's not hard to add back in later if we need it